src/scss/skin-legacy/components/_volumecontrolbutton.scss

```diff
@@ -20,7 +20,7 @@
       bottom: .5em;
       height: auto;
       left: .3em;
-      overflow: hidden;
+      overflow-x: hidden;
```

Comment:
it's required to hide all overflow, else I can sometimes see scroll bars (in Chrome)